tools/run-clang-format-in-ci.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -ex
+
+clang-format --version
+
+# Run clang-format.
+# Exit with non-zero status if output is non-empty.
+[ -z "$(python tools/clang_format.py)" ]
```

Comment:
It would be better to just make `tools/clang_format.py` correctly report status. Does this line even show the output at all? (If it doesn't, that's bad right?)